NOnion/NOnion.fsproj

```diff
@@ -38,12 +38,6 @@
     <Compile Include="TorStream.fs" />
   </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="FSharp.Control.Reactive" Version="5.0.2" />
-    <PackageReference Include="FSharpx.Async" Version="1.14.1" />
-    <PackageReference Include="System.Reactive" Version="5.0.0" />
-  </ItemGroup>
-  
```

Comment:
these dependencies are being added in the 1st commit, but it should be added only on the commit that adds system.reactive support

Reply:
fixed